extensions/data-plane-transfer/data-plane-transfer-sync/src/main/java/org/eclipse/dataspaceconnector/transfer/dataplane/sync/DataPlaneTransferSyncExtension.java

```diff
@@ -87,6 +89,14 @@ public class DataPlaneTransferSyncExtension implements ServiceExtension {
     @Inject
     private PrivateKeyResolver privateKeyResolver;
 
+    @Inject
+    private DataEncrypter dataEncrypter;
+
+    @Provider(isDefault = true)
+    public DataEncrypter getDataEncrypter() {
+        return new NoopDataEncrypter();
```

Comment:
Since we're in the loop I would add a warning saying like "No DataEncrypter registered, a no-op implementation will be used, not suitable for production environments"

Reply:
Good input! I will add this later